templates/ReactReduxSpa/package.json

```diff
@@ -1,7 +1,18 @@
 {
-  "name": "WebApplicationBasic",
+  "name": "react-redux-spa",
   "version": "0.0.0",
   "dependencies": {
```

Comment:
Not sure why I can't comment on line 40 (it's grayed out), but bump typescript to v2.
`"typescript": "^2.0.3",`